npm/ipsm/ipsm_test.go

```diff
@@ -90,16 +96,23 @@ func TestAddToList(t *testing.T) {
 }
 
 func TestDeleteFromList(t *testing.T) {
-	ipsMgr := NewIpsetManager()
-	if err := ipsMgr.Save(util.IpsetTestConfigFile); err != nil {
-		t.Errorf("TestDeleteFromList failed @ ipsMgr.Save")
+	var calls = []testutils.TestCmd{
```

Comment:
better to use string variables to define "test-set" and "test-list" just in case we want to use other values later?

Reply:
discussed offline, will optimize in subsequent pr's